<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OmgLayerSetPageBase</class>
 <widget class="QWizardPage" name="OmgLayerSetPageBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="title">
   <string>Model creation layers or layerset</string>
  </property>
  <property name="subTitle">
   <string>To build your model(s), openLayers needs to use independent variable data such as rainfall, temperature, terrain etc. You can use raster layers loaded in QGIS or a predefined layerset.</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QRadioButton" name="radUseQGISRasters">
     <property name="text">
      <string>Run my model using these layers</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QListWidget" name="lstRasterLayers">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
     <property name="isWrapping" stdset="0">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QRadioButton" name="radUseLayerset">
     <property name="text">
      <string>Run my model using this layerset</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QComboBox" name="cboModelLayerSet">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolModelSettings">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resources/resources.qrc">
         <normaloff>:/configure.png</normaloff>:/configure.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>radUseQGISRasters</sender>
   <signal>toggled(bool)</signal>
   <receiver>lstRasterLayers</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>47</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radUseQGISRasters</sender>
   <signal>toggled(bool)</signal>
   <receiver>cboModelLayerSet</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radUseQGISRasters</sender>
   <signal>toggled(bool)</signal>
   <receiver>toolModelSettings</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>383</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radUseLayerset</sender>
   <signal>toggled(bool)</signal>
   <receiver>lstRasterLayers</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel">
     <x>70</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radUseLayerset</sender>
   <signal>toggled(bool)</signal>
   <receiver>cboModelLayerSet</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>121</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>118</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radUseLayerset</sender>
   <signal>toggled(bool)</signal>
   <receiver>toolModelSettings</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
